README.md

```diff
@@ -80,67 +80,76 @@ cd /path/to/{HOLBA_DIR}
 make main
 ```
 
-Notice that this sequence is exemplary and it is possible to selectively run the `install_*.sh` scripts for the components that are desired. The script `config.env.sh` is generated and contains all variables for components which can be found in `${HOLBA_OPT_DIR}` or are available in the shell when `./configure.sh` runs. Sourcing the script `./env.sh` in the respective copy of the HolBA repository will setup the currently running shell for development there.
+Notice that this sequence is just an example, and it is possible to selectively run the `install_*.sh` scripts for the components that are desired. The script `config.env.sh` is generated and contains all variables for components which can be found in `${HOLBA_OPT_DIR}` or are available in the shell when `./configure.sh` runs. Sourcing the script `./env.sh` in the respective copy of the HolBA repository will setup the currently running shell for development there.
 
 
 
 ## Folders and organization
 
 ```
-├─ doc
-├─ src
-│  ├─ theory
-│  │  ├─ bir: core BIR language
-│  │  ├─ bir-support: various supporting theories for bir
-│  │  ├─ models: various machine models
-│  │  └─ tools: theories used by the tool libraries in src/tools
-│  │     ├─ ...
-│  │     ...
-│  ├─ shared: general BIR libraries, used by libraries in tools
-│  └─ tools
-│     ├─ cfg: Control Flow Graph utilities
-│     ├─ exec: concrete execution
-│     ├─ lifter: proof-producing binary lifter
-│     ├─ pass: Passification utility
-│     ├─ wp: weakest precondition propagation
-│     └─ scamv: abstract side channel model validation framework
-└──── examples: to showcase HolBA
+├─ doc: Documentation material
+├─ examples: Showcasing HolBA
+│  ├─ aes: lifting + WP of AES on Cortex-M0
+│  ├─ bsl-wp-smt: Simplified BIR generation and SMT interface
```

Comment:
It does also have wp generation inside. In this way it tests the whole chain for simple example programs. Maybe we want to add one word somewhere to capture this fact?

Reply:
> It does also have wp generation inside. In this way it tests the whole chain for simple example programs. Maybe we want to add one word somewhere to capture this fact?

Sure, you can suggest a change to this effect.

Reply:
Not exactly one extra word, but what about:
Small BIR example programs to test Simplified BIR generation, WP propagation and SMT interface
or
Small BIR example programs to test BSL, WP, and the extended HolSMT
or append "libraries together" to the last one

Reply:
I went with the first suggestion.